app/controllers/prayer_requests_controller.rb

```diff
@@ -85,15 +85,38 @@ def set_prayer_request
   end
 
   # Only allow a list of trusted parameters through.
-  def prayer_request_params_create
+  def create_prayer_requests_params
```

Comment:
This naming scheme is much better. Verb + object follows our implicit naming standards.